src/prefect/runner/utils.py

```diff
@@ -90,3 +97,31 @@ def update_refs_to_components(openapi_schema: Dict[str, Any]) -> Dict[str, Any]:
         update_refs_in_schema(definition, "#/components/schemas/")
 
     return openapi_schema
+
+
+async def _find_subflows_of_deployment(
+    deployment: "Deployment",
+) -> List[Tuple[str, "Flow"]]:
+    """
+    Find all subflows of a deployment.
+
+    Args:
+        deployment: The deployment to find subflows of.
+
+    Returns:
+        A list of flows.
+    """
+
+    entrypoint_dir = os.path.dirname(deployment.entrypoint.split(":")[0])
+
+    flow_entrypoints = [
+        entrypoint
+        for flow in await _search_for_flow_functions(entrypoint_dir)
```

Comment:
Why is it that we'd want to omit the deployment's entrypoint? Are we taking that to be the-main-flow-and-definitely-not-a-subflow?

Reply:
to surface some sync discussion on this:

- this was to avoid adding a `/flow/...` endpoint for flows that are deployment entrypoints and thus already have a `/deployment/...` endpoint